awx/ui_next/src/components/FormField/TextAndCheckboxField.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useField } from 'formik';
+import { t } from '@lingui/macro';
+import {
+  FormGroup,
+  TextInput,
+  Button,
+  InputGroup as PFInputGroup,
+} from '@patternfly/react-core';
+import PFCheckIcon from '@patternfly/react-icons/dist/js/icons/check-icon';
+import styled from 'styled-components';
+import Popover from '../Popover';
+
+const InputGroup = styled(PFInputGroup)`
+  padding-bottom: 5px;
+`;
+const CheckIcon = styled(PFCheckIcon)`
+  color: var(--pf-c-button--m-plain--disabled--Color);
+  ${props =>
+    props.isSelected &&
+    `color: var(--pf-c-button--m-secondary--active--Color)`};
+`;
+
+const validate = () => {
+  return value => {
+    let message;
+    const hasValue = value.find(({ choice }) =>
+      choice.trim().length > 0 ? choice : undefined
+    );
+    if (!hasValue) {
+      message = t`There must be a value in at least one input`;
+    }
+    return message;
+  };
+};
+function TextAndCheckboxField({ label, tooltip }) {
+  const [
+    formattedChoicesField,
+    formattedChoicesMeta,
+    formattedChoicesHelpers,
+  ] = useField({
+    name: 'formattedChoices',
+    validate: validate(),
+  });
+
+  const [typeField] = useField('type');
+  const isValid = !(formattedChoicesMeta.touched && formattedChoicesMeta.error);
+
+  return (
+    <FormGroup
+      label={label}
+      isRequired
+      helperText={
+        !formattedChoicesField.value[0].choice.trim().length
+          ? t`Type answer then click checkbox on right to select answer as default.`
+          : t`Press 'Enter' to add more answer choices. One answer choice per line. `
+      }
+      helperTextInvalid={formattedChoicesMeta.error}
+      onBlur={e => {
+        if (!e.currentTarget.contains(e.relatedTarget)) {
+          formattedChoicesHelpers.setTouched();
+        }
+      }}
+      validated={isValid ? 'default' : 'error'}
+      labelIcon={<Popover content={tooltip} />}
+    >
+      {formattedChoicesField.value.map(({ choice, isDefault }, i) => (
+        <InputGroup>
+          <TextInput
+            aria-label={choice}
```

Comment:
I'm reluctant to set a default value for a new choice because I don't want the user to accidentally submit the default choice, but I can set an aria-label